src/Padawan/Domain/Completer/GlobalFunctionsCompleter.php

```diff
@@ -44,13 +44,28 @@ public function getEntries(Project $project, Context $context)
 
     public function canHandle(Project $project, Context $context)
     {
-        $postfix = "";
-        if (is_string($context->getData())) {
-            $postfix = trim($context->getData());
-        }
+        $postfix = $this->getPostfix($context);
         return parent::canHandle($project, $context)
             && ($context->isString() || $context->isEmpty())
             && strlen($postfix) > 0
             ;
     }
+
+    private function getPostfix(Context $context)
+    {
+        if (is_string($context->getData())) {
+            return trim($context->getData());
+        }
+        if (empty($postfix)) {
+            $contextData = $context->getData();
+            if (is_array($contextData) && @$contextData[3] instanceof \PhpParser\Node\Arg) {
```

Comment:
To replace the logic, the deepest layer possible may be [the ContextResolver class](https://github.com/halftan/padawan.php/blob/support-completion-funcs-as-method-params/src/Padawan/Framework/Complete/Resolver/ContextResolver.php#L76).